docs/configuration-reference/components/azure-arc-onboarding.md

```diff
@@ -0,0 +1,119 @@
+[Linux](https://azure.microsoft.com/en-in/overview/linux-on-azure/) servers, SQL server,
+[Kubernetes](https://azure.microsoft.com/en-in/services/kubernetes-service/) clusters and [Azure
+services](https://azure.microsoft.com/en-in/services/azure-arc/hybrid-data-services/).
+
+- Establish central visibility in the [Azure portal](https://azure.microsoft.com/en-in/features/azure-portal/)
+and enable multi-environment search with Azure Resource Graph.
+
+- Meet [governance](https://azure.microsoft.com/en-in/solutions/governance/) and compliance standards for
+apps, infrastructure and data with [Azure Policy](https://azure.microsoft.com/en-in/services/azure-policy/).
+
+- Delegate access and manage security policies for resources using role-based access control (RBAC) and [Azure
+Lighthouse](https://azure.microsoft.com/en-in/services/azure-lighthouse/).
+
+- Organise and inventory assets through a variety of Azure scopes, such as management groups, subscriptions,
+resource groups and tags.
+
+This component onboards or removes a Lokomotive cluster with Azure Arc.
+
+## Prerequisites
+
+1. Register providers for Azure Arc enabled Kubernetes:
+   ```bash
+   az provider register --namespace Microsoft.Kubernetes
+   az provider register --namespace Microsoft.KubernetesConfiguration
+   az provider register --namespace Microsoft.ExtendedLocation
+   ```
+
+   Monitor the registration process. Registration may take up to 10 minutes.
+
+   ```bash
+   az provider show -n Microsoft.Kubernetes -o table
+   az provider show -n Microsoft.KubernetesConfiguration -o table
+   az provider show -n Microsoft.ExtendedLocation -o table
+   ```
+
+   **NOTE**: This registration is needed only once per tenant.
+
+2. Create a resource group
+
+   Example:
+
+   ```bash
+   az group create --name AzureArcTest --location EastUS --output table
+   ```
+
+3. Register an application with the Microsoft identity platform
+
+   a. Register an application by following the steps outlined in the [Azure
+   documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#register-an-application).
+
+   b. Add credentials for the registered application by following the steps outlined
+[here](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app#add-a-client-secret).
+
+      Take note of the Application ID, Tenant ID, Secret ID and Secret Value, in the overview section.
+      You'll need them in the component configuration.
+
+   c. Provide Azure Arc related permissions to the Service principal object(App registration). [Azure
+   documentation](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal)
+      provides a guided way to add permissions to the Service principal.
+
+   The following set of roles are needed
+     * Kubernetes Cluster - Azure Arc Onboarding
+     * Microsoft.Kubernetes connected cluster role
```

Comment:
This seems like a lot of description for reference page. I'd expect it to be more concise. Maybe it would make sense to create a guide for it or something?

Speaking from experience, mixing "guide" content with reference documentation makes it harder to find both information, as usually "guide" is to follow and get something specific to work, while with configuration reference, you either look for specific key or you scan the index (keys, properties) to find one relevant to your problem. Having a lot of text in the middle makes it more difficult.

Perhaps we can have a short list of prerequisites here, like for other components and link to the beginning of the guide for more details?

Reply:
In the prerequisites section of the configuration reference document   instructions are now replaced with ha link to the how-to guide prerequisites section

Reply:
Looks good, thanks!